src/utils/filter.js

```diff
@@ -0,0 +1,14 @@
+import {FilterType} from '../constants.js';
+import {
+  isEventPointInFuture,
+  isEventPointInPast
+} from './event-point.js';
+
+
+const filter = {
+  [FilterType.EVERYTHING] : (eventPoints) => eventPoints.filter((point) => point),
```

Comment:
Можно не фильтровать
```suggestion
  [FilterType.EVERYTHING] : (eventPoints) => eventPoints,
```